src/main/scala/org/broadinstitute/dsde/workbench/sam/google/GoogleKeyCache.scala

```diff
@@ -21,13 +25,15 @@ import scala.concurrent.{ExecutionContext, Future}
 class GoogleKeyCache(
     val googleIamDAO: GoogleIamDAO,
     val googleStorageDAO: GoogleStorageDAO,
```

Comment:
do we still need the old `googleStorageDAO`?

Reply:
it's not directly used by `GoogleKeyCache` anymore. But it's referenced in `GoogleKeyCacheMonitorSupervisor`. I guess I can pass the old `googleStorageDAO` directly to `GoogleKeyCacheMonitorSupervisor`